<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotConsoleWindow</class>
 <widget class="QMainWindow" name="RobotConsoleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="openGLviewLayout"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="consoleOutput">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout"/>
    </item>
    <item>
     <widget class="QLineEdit" name="commandLine">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTitle">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuTitle"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>commandLine</sender>
   <signal>returnPressed()</signal>
   <receiver>RobotConsoleWindow</receiver>
   <slot>commandEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>commandEntered(QString)</slot>
  <slot>resetPressed()</slot>
 </slots>
</ui>
